README.md

```diff
@@ -255,9 +255,96 @@ export HUGGING_FACE_HUB_TOKEN=your-auth-token
 
 and uncomment the line containing `--hf_repo_for_upload ...` in the above call to `inference/convert_composer_to_hf.py`.
 
-# :construction: UNDER CONSTRUCTION: Registry
+# Registry
+
+Some components of LLM Foundry are registrable. This means that you can register options for these components, and then use them in your yaml config, without forking the library.
```

Comment:
how about something like:

```suggestion
You can use the registry to customize your workflows without forking the library. Some components of LLM Foundry are registrable, such as models, loggers, and callbacks. This means that you can register new options for these components, and then use them in your yaml config.
```

Wanted to mention examples of components (loggers, models, callbacks) and explain that its for customization.